tools/list-missing-regions-and-languages.sh

```
@@ -6,7 +6,7 @@ echo "----------------------------------------------------------------------"
 echo "Missing territories:"
 MISSING_TERRITORIES=
 MISSING_TERRITORIES_COUNT=0
-for i in $(locale -a | grep -a _ | perl -pe 's/.*_([A-Z]{2,2}).*/\1/g' | sort | uniq  ); do grep -q $i territories.xml; if [ $? -eq 1 ]; then MISSING_TERRITORIES="$MISSING_TERRITORIES $i"; MISSING_TERRITORIES_COUNT=$(expr $MISSING_TERRITORIES_COUNT + 1); fi; done
+for i in $(locale -a | grep -a _ | perl -pe 's/[a-z]{2,3}_([A-Z]{2,2}).*/\1/g' | sort | uniq  ); do grep -q $i territories.xml; if [ $? -eq 1 ]; then MISSING_TERRITORIES="$MISSING_TERRITORIES $i"; MISSING_TERRITORIES_COUNT=$(expr $MISSING_TERRITORIES_COUNT + 1); fi; done
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)